content/_index.md

```diff
@@ -27,6 +27,86 @@ If you're new to KES you may want to start in one of the following pages:
+Each policy consists of one or more identities, where each identity corresponds to the SHA-256 hash of an x.509 certificate. 
+The server only allows the client to perform the requested operation if *all* of the following are true:
+
+- A policy on the KES server contains the client identity.
+- The policy explicitly allows the requested operation.
+
+If no such policy exists on the KES server *or* if the policy does not explicitly allow the requested operation, the KES server denies the client's request.
+
+### KES Policies
+
+KES uses policy-based access control (PBAC), where a policy describes the operations which an authenticated client may perform. 
+
+The following `YAML` document provides an example of the :kesconf:`policy` section of the KES server configuration document. 
+The policy `minio-sse` includes the appropriate :API endpoints for supporting MinIO Server-Side Encryption:
+
+```yaml {.copy}
+policy:
+   minio-sse:
+      paths:
+      - /v1/key/create/*
+      - /v1/key/generate/*
+      - /v1/key/decrypt/*
+      - /v1/key/delete/*
+      identities:
+      - <SHA-256 HASH>
+```
+
+- Each element in the `policy.policyname.paths` array represents an `API endpoint` to which the policy grants access.
+
+- Each element in the `policy.policyname.identities` array represents the SHA-256 hash of an x.509 certificate presented by a client.
+
+  Use the [`kes identity of`]({{< relref "cli/kes-identity/of.md" >}}) command to compute the identity hash of a client's x.509 certificate. 
+
+Policies and identities have a one-to-many relationship, where one policy can contain many identities. 
+*However*, a given identity can associate to at most one policy at a time.
+
+The KES server provides two methods for configuring policies:
+
+- The `policy` section of the KES [configuration file]({{< relref "tutorials/configuration.md#config-file" >}}) lists the persistent policies for the KES server.
+
+- The [`kes policy`]({{< relref "cli/kes-policy/_index.md" >}}) command supports creating *ephemeral* policies for the KES server. 
+  The [`kes identity`]({{< relref "cli/kes-identity/_index.md" >}}) command supports *ephemeral* modification of the identities associated to policies on the KES server.  
+
+  Policies created or modified using either `kes policy` or `kes identity` disappear after restarting the KES server.
+
+{{< admonition type="note" >}}
+Each KES server has its own configuration file from which it derives all persistent policies. 
+With distributed KES deployments, each server has its own independent and distinct policy-identity set based on its configuration file. 
+This may allow for an identity to associate to different policies depending on which KES server a client connects to.
+
+Exercise caution in allowing KES servers to contain differing policies, as it may result in inconsistent client encryption behavior between servers.
```

Comment:
Is "allowing KES servers to contain differing policies" a thing that randomly happens? Or is this more like "avoid configuring different policies for different servers." (Not sure how strongly it needs to be avoided.)